wperf-test/wperf-test-spe_device.cpp

```diff
@@ -91,5 +91,19 @@ namespace wperftest
 			Assert::IsTrue(spe_device::is_filter_name_alias(L"b"));
 			Assert::IsTrue(spe_device::is_filter_name_alias(L"ts"));
 		}
+
+		TEST_METHOD(test_spe_device_get_filter_name)
+		{
+			Assert::AreEqual(spe_device::get_filter_name(L"ld").c_str(), L"load_filter");
+			Assert::AreEqual(spe_device::get_filter_name(L"st").c_str(), L"store_filter");
+			Assert::AreEqual(spe_device::get_filter_name(L"b").c_str(), L"branch_filter");
+			Assert::AreEqual(spe_device::get_filter_name(L"ts").c_str(), L"ts_enable");
+			Assert::AreEqual(spe_device::get_filter_name(L"min").c_str(), L"min_latency");
+
+			uint64_t val = SPE_CTL_FLAG_VAL_MASK;
+
+			Assert::AreEqual(spe_device::max_filter_val(L"min"), val);
+			Assert::AreEqual(spe_device::max_filter_val(L"min_latency"), val);
```

Comment:
1) This should be a separate set of tests, this one tests `get_filter_name()`.
2) Also, less code:
```suggestion
			Assert::AreEqual(spe_device::max_filter_val(L"min"), SPE_CTL_FLAG_VAL_MASK);
			Assert::AreEqual(spe_device::max_filter_val(L"min_latency"), SPE_CTL_FLAG_VAL_MASK);
```